unittests/tools/test_anchore_grype_parser.py

```diff
@@ -86,3 +88,165 @@ def test_grype_parser_with_new_matcher_list(self):
                 self.assertEqual("3.2.0", finding.component_version)
                 found = True
         self.assertTrue(found)
+
+    def test_check_all_fields(self):
+        testfile = open("unittests/scans/anchore_grype/check_all_fields.json")
+        parser = AnchoreGrypeParser()
+        findings = parser.get_findings(testfile, Test())
+        testfile.close()
+        self.assertEqual(5, len(findings))
+
+        finding = findings[0]
+        self.assertEqual('CVE-2004-0971 in libgssapi-krb5-2:1.17-3+deb10u3', finding.title)
+        description = '''**Vulnerability Id:** CVE-2004-0971
+**Vulnerability Namespace:** debian:10
+**Related Vulnerability Description:** The krb5-send-pr script in the kerberos5 (krb5) package in Trustix Secure Linux 1.5 through 2.1, and possibly other operating systems, allows local users to overwrite files via a symlink attack on temporary files.
+**Matcher:** dpkg-matcher
+**Package URL:** pkg:deb/debian/libgssapi-krb5-2@1.17-3+deb10u3?arch=amd64'''
+        self.assertEqual(description, finding.description)
+        self.assertEqual('CVE-2004-0971', finding.cve)
+        self.assertEqual(1352, finding.cwe)
+        self.assertIsNone(finding.cvssv3)
+        self.assertIsNone(finding.cvssv3_score)
+        self.assertEqual('Info', finding.severity)
+        self.assertIsNone(finding.mitigation)
+        references = '''**Vulnerability Datasource:** https://security-tracker.debian.org/tracker/CVE-2004-0971
+**Related Vulnerability Datasource:** https://nvd.nist.gov/vuln/detail/CVE-2004-0971
+**Related Vulnerability URLs:**
+- http://www.securityfocus.com/bid/11289
+- http://www.gentoo.org/security/en/glsa/glsa-200410-24.xml
+- http://www.redhat.com/support/errata/RHSA-2005-012.html
+- http://www.trustix.org/errata/2004/0050
+- http://bugzilla.redhat.com/bugzilla/show_bug.cgi?id=136304
+- https://exchange.xforce.ibmcloud.com/vulnerabilities/17583
+- https://oval.cisecurity.org/repository/search/definition/oval%3Aorg.mitre.oval%3Adef%3A10497
+- https://lists.apache.org/thread.html/rc713534b10f9daeee2e0990239fa407e2118e4aa9e88a7041177497c@%3Cissues.guacamole.apache.org%3E'''
+        self.assertEqual(references, finding.references)
+        self.assertEqual('libgssapi-krb5-2', finding.component_name)
+        self.assertEqual('1.17-3+deb10u3', finding.component_version)
+        self.assertEqual('CVE-2004-0971', finding.vuln_id_from_tool)
+        self.assertEqual(['dpkg'], finding.tags)
```

Comment:
better use:
```suggestion
        self.assertIn('dpkg', finding.tags)
```
More future proof if we add more tags

Reply:
The idea is to check exactly what's in the test data. If someone changes the test data without the test case, I think it's ok when it fails.

Reply:
ok